src/nvme/ioctl.c

```diff
@@ -112,8 +112,136 @@ static void nvme_show_command(struct nvme_passthru_cmd *cmd, int err, struct tim
 	printf("timeout_ms   : %08x\n", cmd->timeout_ms);
 	printf("result       : %08x\n", cmd->result);
 	printf("err          : %d\n", err);
-	printf("latency      : %lu us\n",
-	       (end.tv_sec - start.tv_sec) * 1000000 + (end.tv_usec - start.tv_usec));
+}
+
+const char *nvme_admin_to_string(__u8 opcode)
```

Comment:
Would these functions be useful to export from `libnvme`?

Reply:
Thanks for your comments. Yes the code basically copied from `nvme-cli: nvme_cmd_to_string()` and duplicated so I will do merge the functionality to export from `libnmve`.